hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -345,8 +345,10 @@ private function up1402_removedUnusedManagedEntities($caseTypes, $activityTypes)
       $entitiesToRemove[$caseType] = 'CaseType';
     }
 
-    foreach ($activityTypes as $activityType) {
-      $entitiesToRemove['civitask:act:' . $activityType] = 'OptionValue';
+    foreach ($activityTypes as $extension) {
+      foreach ($extension as $activityType) {
+        $entitiesToRemove['civitask:act:' . $activityType] = 'OptionValue';
+      }
```

Comment:
Just a question, since HRCase doesn't seem to have any tests in it did you find this bug from another test failing?